### PR TITLE
Con 2768 favorites toggle on list v2

### PR DESCRIPTION
**Pull request by sznowicki**

### CODE REVIEW

File: favorites/reducers/products.js [hunk 1]
```diff
@@ -26,65 +25,71 @@ import {
...
         ...state,
-        isFetching: false,
-        ids: [
-          ...state.ids,
-          action.productId,
-        ],
+        isFetching: true,
+        ids: state.ids || [],
+        expires: 0,
+        // No ready here! It should be undefined or true!
```
Comment (CarinaHoffmann) [inline favorites/reducers/products.js]: What is this comment supposed to mean?

File: favorites/actions/toggleFavorites.js [hunk 2]
```diff
@@ -7,102 +7,93 @@
...
-      .catch(rej);
-  });
-
-  removePromise
-    .then(action => dispatch(action))
-    .catch(() => dispatch(errorRemoveFavorites(productId)));
-
-  return removePromise;
+const removeFavorites = (productId, withRelatives = false) => (dispatch, getState) => {
+  // Temporary solution
```
Comment (CarinaHoffmann) [inline favorites/actions/toggleFavorites.js]: Unnecessary comment.

File: favorites/action-creators/index.js [hunk 3]
```diff
@@ -21,14 +27,36 @@ export const errorFetchFavorites = error => ({
...
+ * @param {string} productId Product identifier.
+ * @returns {Object}
+ */
+export const requestAddFavorites = productId => ({
+  type: REQUEST_ADD_FAVORITES,
+  productId,
+});
+
+/**
+ * Request add favorites action.
```
[the remark this answers is not in the document]
Comment (sznowicki) [reply favorites/action-creators/index.js]: Done.

File: favorites/reducers/products.js (hunk 1 above)
Comment (sznowicki) [reply favorites/reducers/products.js]: Updated.

File: favorites/actions/toggleFavorites.js (hunk 2 above)
Comment (sznowicki) [reply favorites/actions/toggleFavorites.js]: Removed.

**Pull request merged**